### PR TITLE
Kubernetes Instructions

### PR DESCRIPTION
**Pull request by triceam**
I added instructions in `README.md` to quickly target the Kubernetes environment on Bluemix using the IBM Cloud Developer Tools CLI, and one minor modification to `package.json` so that it is compatible with `bx dev debug` commands from the CLI.  

This uses the same steps as running locally, and uses `bx dev enable` to generate Kubernetes assets, with `bx dev deploy` to handle compilation of the Docker image, push to IBM Container Registry, and the subsequent push to Kubernetes using the generated helm chart.

This does not cover Watson service bindings to Kubernetes clusters, as that requires more changes to the application code, and is not fully supported by the Cloud Developer Tools CLI *yet*.

### CODE REVIEW

**Comment by scottdangelo** [comment]
This looks good. We're working through the details of adding Bluemix Container (K8s) instructions for another journey ([watson-convesation-slots-intro](https://github.com/IBM/watson-conversation-slots-intro)) and we'll be adding BX dev tools CLI instructions there as well. After we've worked out the details of where the instructions will live in the repo, and how they will be linked in the top-level README, we'll revisit this PR and make them consistent.
Thanks.

File: README.md [hunk 1]
```diff
@@ -59,6 +59,100 @@ Use the ``Deploy to Bluemix`` button **OR** create the services and run locally.
...
+    curl -sL https://ibm.biz/idt-installer | bash
+    ```
+
+    If you're on a windows machine, open Windows PowerShell by right-clicking and select "Run as Administrator":
+
+    ```
+    Set-ExecutionPolicy Unrestricted; iex(New-Object Net.WebClient).DownloadString('http://ibm.biz/idt-win-installer')
+    ``` 
+
+    This will install the IBM Cloud Developer Tools CLI, the Bluemix CLI, Docker, the Kubernetes CLI, and all other dependencies if they are not already installed.
```
Comment (dolph) [inline README.md]: I don't think "and all other dependencies" is particularly in the spirit of journeys. Ideally, the developer should be able to skip a script that does all the work in favor of doing as much of it as possible using documentation alone. In this case, I had to go read exactly what https://ibm.biz/idt-installer actually installs in otinstalls in order to follow along.

**Comment by triceam** [comment]
Just curious, what's the status here?  Language and marketing teams are pushing the video that goes along with this, but the instructions aren't part of the repo yet b/c this pr isn't merged.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,100 @@ Use the ``Deploy to Bluemix`` button **OR** create the services and run locally.
...
+
+        ```
+        bx cr namespace-add <my_namespace>
+        ```
+        
+      This will add a namespace to create your own image repository. Replace `<my_namespace>` with your preferred namespace.
+
+4. Follow all 6 steps in the instructions below to [run locally](#run-locally).
+
+5. Open a terminal and cd into the folder containing the project downloaded in the `run locally` steps.   Within a terminal use the `bx dev enable` command and answer all questions to generate Kubernetes deployment assets.
```
Comment (dolph) [inline README.md]: Again, in the spirit of a journey, "and answer all questions" really needs to be expanded upon. The questions are effectively non-trivial for a newcomer, and I would expect most of them to be answered incorrectly as a result.

**Comment by scottdangelo** [comment]
We've put k8s instructions for conversation-slots into a separate doc:
https://github.com/IBM/watson-conversation-slots-intro/blob/master/doc/source/Container.md
and linked to it from the README.
So, whatever revisions are required for these instructions, they'd need to be put into a doc/sources/*.md and linked to from the README>

**Comment by markstur** [comment]
Closing. This is out-of-date now. Sorry it got held up in review. We're doing OpenShift instructions now so I'm not sure we'll need to add k8s instructions too.

**Pull request closed without merging**